src/viminfo.c

```diff
@@ -2895,17 +2905,18 @@ read_viminfo_up_to_marks(
 /*
  * do_viminfo() -- Should only be called from read_viminfo() & write_viminfo().
  */
-    static void
+    static int
```

Comment:
What is the reason for returning a return value here? It seems like all callers of `do_viminfo()` do not check the return value, so I guess it is no needed?